index.d.ts

```diff
@@ -2038,7 +2010,6 @@ declare namespace Shopify {
   interface IOrderTaxLine {
     title: string;
     price: string;
-    price_set: IOrderAdjustmentAmountSet;
```

Comment:
Added in https://github.com/MONEI/Shopify-api-node/commit/f71a10a3b588d1e889d792188dcc498d1afa301e I don't think it should be removed.

Reply:
At that time, this `IOrderTaxLine` type was used for both order-level `tax_lines` (which don’t have `price_set`) and item-level `tax_lines` (which do have `price_set`). This change reorganizes types so that this is only used for order-level ones, so this should not have `price_set`.